qutebrowser/browser/urlmarks.py

```diff
@@ -53,43 +53,25 @@ class InvalidUrlError(Error):
     pass
 
 
-class DoesNotExistError(Error):
```

Comment:
To expand on this: I think it's fine to use `KeyError` on a lower level, i.e. with the `SqlTable` object with a dict-like interface. However, on this higher level, I think it makes sense to re-raise them as more specific exceptions.